app/modules/softwareupdate/softwareupdate_model.php

```diff
@@ -0,0 +1,146 @@
+<?php
+class Softwareupdate_model extends Model 
+{
+        public function __construct($serial='')
+        {
+                parent::__construct('id', 'softwareupdate'); //primary key, tablename
+                $this->rs['id'] = 0;
+                $this->rs['serial_number'] = $serial; $this->rt['serial_number'] = 'VARCHAR(255) UNIQUE';
+                $this->rs['automaticcheckenabled'] = 0; $this->rt['automaticcheckenabled'] = 'INT';
```

Comment:
No need to specify INT, MR will do that automatically based on the initial value.